File: Week1/05-03/printBox.js
/*
## Print a Box

Given a height and width, print a box consisting of * characters as its border. For example, given the height of 4 and the width of 6, you should print:

```
******
*    *
*    *
******
```

#loop #if #medium
*/

var height = 10;
var width = 20;
var row = "";

for (var i = 1; i <= height; i++) {
  for (var j = 1; j <= width; j++) {
    if (i === 1 || i === height) {
      row = row + "*";
    }
    else if (j === 1 || j === width){
      row = row + "*";
    }
    else {
      row = row + " ";
    }
  }
  console.log(row);
  row = "";
}

File: Week11/07-13/print-box.py
## Print a Box

# Given a height and width, print a box consisting of * characters as its border. You will prompt the user to enter the height and width. For example, given the height of 4 and the width of 6, you should print:
#
# ```
# ******
# *    *
# *    *
# ******
# ```

print "Please enter a height for the box:"
height = int(raw_input('> '))

print "Please enter a width for the box:"
width = int(raw_input('> '))

for x in range(1, height + 1):
    if x == 1 or x == height:
        print '*' * width
    else:
        spaces = ' ' * (width - 2)
        print '*' + spaces + '*'
